articles/custom-domains/feature-specific-setup.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Feature Specific Setup with Custom Domains
+description: How to set up custom domains with feature specific instructions
+toc: true
+---
+# Feature-Specific Setup
+
+There are additional steps you must complete depending on which Auth0 features you are using.
+
+::: warning
+If you have been using Auth0 for some time and decide to enable a custom domain, you will have to migrate your existing apps and update the settings as described below. Note that existing sessions created at `tenant.auth0.com` will no longer be valid once you start using your custom domain, so users will have to login again.
+:::
+
+## Configure the Login Page
+
+When using custom domains with [universal login](/hosted-pages/login), you will need to determine which of the following apply to you:
+
+* If you're using the **default** login page without customization, you will not need to make any changes. Your custom domain will work right out of the box.
```

Comment:
I'd skip replace the first sentence with the first bullet of this section. `If you use the default login page, without customization, then you do not need to make any changes. Otherwise you need to make the following changes in your code:` (sth like that, same info, less words)